### PR TITLE
Fix eap peap tls on android

### LINKED ISSUE #5966: EAP-PEAP configuration for android api > 29
**Describe the bug**
For API>29, the PaketFence Android Agent needs the CA certificate public key that has signed the radius certificate.

**To Reproduce**
Use an android device with API 30 (Android 11)
Use the android agent 2.4.0 from Google Playstore
Use Radius self-signed certificate
Use android Provisioner (with pki settings etc.)
Try EAP PEAP connection.
=> The current Android application crash due to this test:
https://cs.android.com/android/platform/superproject/+/master:frameworks/base/wifi/java/android/net/wifi/WifiEnterpriseConfig.java;l=1426;drc=master

**Expected behavior**
EAP PEAP Connection trough PacketFence-Android-Agent for Android API > 29 (more than android 10)

**How to fix**
- Add a new field for the CA public key in Android Provisioner after the "RADIUS server certificate path".
- Add this new field in the lib/pf/provisioner/mobileconfig.pm and where it is needed ;)
- Change eap peap xml file (html/captive-portal/templates/wireless-profile-peap.xml) to get the CA content form the previous field.
- Test if the new eap peap xml is running well with other devices (Apple, Windows, etc)

It has been tested with sucess by changing a xml modified by hand, using an Android 11 and a modified android agent.
In the example bellow, the Android Agent grab the CA with the <key>PayloadType</key> and <string>com.apple.security.ca</string>.

Example:
```
# cat /usr/local/pf/html/captive-portal/templates/wireless-profile-peap.xml 
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<!-- Generated by the iPhone Configuration Utility /-->
<plist version="1.0">
<dict>
        <key>PayloadContent</key>
        <array>
                <dict>
                        <key>AutoJoin</key>
                        <true/>
[...]
                        <key>PayloadType</key>
                        <string>com.apple.wifi.managed</string>
[...]
                </dict>
                <dict>
                        <key>PayloadCertificateFileName</key>
                        <string>[% provisioner.server_certificate_cn() %]</string>
[...]
                        <key>PayloadType</key>
                        <string>com.apple.security.root</string>
[...]
                </dict>
                <dict>
                        <key>PayloadContent</key>
                        [% IF for_windows || for_android %]
<!-- Here there is the raw content of the public ca cert /-->
                        <string>
MIID5DCCAsygAwIBAgIBATANBgkqhkiG9w0BAQsFADBqMQswCQYDVQQGEwJDQTEN
MAsGA1UECBMETm9uZTENMAsGA1UEBxMETm9uZTENMAsGA1UECRMETm9uZTENMAsG
[...]
KU/78Hsljv7g09mkVneRtrOd8hqKK7Mcb3Ycj+xHu4V8vop0p9ri/bIK+tIM+/Ag
z+csGKEght9Mmc6uh/EFdZ6mFDUvYSRAeO9Zcs9yre54lJH7VpADKEn2PeO1PdAS
2ZiPL6mU5kaYE6drG9qCiBBti2aIvxD7ox1SDM+q/GK6H+Ji6Ser+w==
                        </string>
                        [% END %]
                        <key>PayloadType</key>
                        <string>com.apple.security.ca</string>
                </dict>
        </array>
        <key>PayloadDescription</key>
        <string>Profile description.</string>
[...]
        <key>PayloadUUID</key>
        <string>5F3EC12F-570E-48EE-A433-E8DC8E9EC4FE</string>
        <key>PayloadVersion</key>
        <integer>1</integer>
</dict>
</plist>
```

### PR DESCRIPTION
**Pull request by JeGoi**
# Description
This will fix eap peap and eap tls for Android agent (and perhaps windows agent if set).
Eap PEAP, it adds a new field for Radius CA in android provisioner. It make the ca cert content available for the Android app.
EAP TLS, it removes the "if" statement to have a "full" xml available.

# Impacts
Now, Android Agent can now create a eap peap and tls profile.

# Issue
fixes #5966

# Delete branch after merge
YES please

### CODE REVIEW

**Comment by JeGoi** [comment]
I'm not sure that the radius ca field is the best user way because it obliges the admin to create a file with the public radius ca key on the server side. It should be better to populate this from the db or use it directly.
It was faster to test and fix this way but it is not that user friendly...

**Comment by JeGoi** [comment]
To reviewers, I guess `server_radius_ca_path` should have bee named `ca_cert_path` like in [pki provider](https://github.com/inverse-inc/packetfence/blob/devel/html/pfappserver/lib/pfappserver/Form/Config/PKI_Provider/packetfence_pki.pm#L144)

**Comment by nqb** [comment]
I will be happy to tests this PR and to add the documentation part.

**Comment by JeGoi** [comment]
The new field is tested as required from the UI.
Not sure it is good.

File: html/pfappserver/root/static.alt/src/views/Configuration/_config/provisioning.js [hunk 1]
```diff
@@ -834,6 +834,20 @@ export const viewFields = {
       ]
     }
   },
+  ca_cert_path: (form = {}, meta = {}) => {
+    return {
+      label: i18n.t('RADIUS server CA path'),
+      text: i18n.t('The path to the RADIUS server CA which have sign the radius certification.'),
```
Comment (julsemaan) [inline html/pfappserver/root/static.alt/src/views/Configuration/_config/provisioning.js]: The path to the RADIUS server CA which signed the server certificate

Comment (julsemaan) [reply html/pfappserver/root/static.alt/src/views/Configuration/_config/provisioning.js]: You still have bad verbiage (i.e. `which have sign the radius certification` is bad)

Comment (JeGoi) [reply html/pfappserver/root/static.alt/src/views/Configuration/_config/provisioning.js]: Where? Latest commit did not fix it?

Comment (julsemaan) [reply html/pfappserver/root/static.alt/src/views/Configuration/_config/provisioning.js]: Nope still says 'which signed the radius certification'

Comment (JeGoi) [reply html/pfappserver/root/static.alt/src/views/Configuration/_config/provisioning.js]: Oh that is right... I missed the point.

**Pull request merged**